src/openforms/appointments/base.py

```diff
@@ -79,6 +79,12 @@ class BasePlugin(AbstractBasePlugin):
 
     configuration_options = EmptyOptions
 
+    @property
+    def is_enabled(self):
```

Comment:
Since I added this functionality to `AbstractBasePlugin` and `BaseRegistry`, I have to explicitly set this here.